.github/workflows/deploy_production.yml

```diff
@@ -13,8 +13,8 @@ jobs:
     # The type of runner that the job will run on
     runs-on: ubuntu-latest
     env:
-      AWS_S3_BUCKET_NAME: www.noah.ph
-      AWS_CF_DISTRIBUTION_ID: E1J3VDQ3CVKWG2
+      AWS_S3_BUCKET_NAME: noah-frontend
+      AWS_CF_DISTRIBUTION_ID: E2MDZCX9DV1ZCO
```

Comment:
Remove these if you're already using secrets for these variables.

Reply:
Changed variables using secrets.